content/en/docs/tasks/debug-application-cluster/resource-usage-monitoring.md

```diff
@@ -4,19 +4,43 @@ reviewers:
 title: Tools for Monitoring Compute, Storage, and Network Resources
 ---
 
-Understanding how an application behaves when deployed is crucial to scaling the application and providing a reliable service. In a Kubernetes cluster, application performance can be examined at many different levels: containers, [pods](/docs/user-guide/pods), [services](/docs/user-guide/services), and whole clusters. As part of Kubernetes we want to provide users with detailed resource usage information about their running applications at all these levels. This will give users deep insights into how their applications are performing and where possible application bottlenecks may be found.
+To scale and application and provide a reliable service, you need to
+understand how an application behaves when it is deployed. You can examine
+application performance in a Kubernetes cluster by examining the containers,
+[pods](/docs/user-guide/pods), [services](/docs/user-guide/services), and
+the characteristics of the overall cluster. Kubernetes provides detailed
+information about an application's resource usage at each of these levels.
+This information allows you to evaluate your application's performance and
+where bottlenecks can be removed to improve overall performance.
 
 ## Overview
 
-Kubernetes monitoring is designed to be monitoring-solution agnostic.  On many clusters, monitoring is provided via two separate pipelines: the resource metrics pipeline, and full monitoring pipeline.
-The resource metrics pipeline provides a limitted set of metrics to cluster components such as the HorizontalPodAutoscaler controller, and `kubectl top`.  These metrics are generally collected by
-[metrics-server](https://github.com/kubernetes-incubator/metrics-server), and exposed via the `metrics.k8s.io` API.  metrics-server discovers all nodes on the cluster, and queries their
-[Kubelet](/docs/admin/kubelet)s for CPU and memory usage.  The Kubelet itself fetches the data from [cAdvisor](https://github.com/google/cadvisor).
+In Kubernetes, application monitoring does not depend on a single monitoring
+solution. On new clusters, you can use two separate pipelines to collect
+monitoring statistics by default:
+
+- The **resource metrics pipeline** provides a limited set of metrics related
+  to cluster components such as the HorizontalPodAutoscaler controller, as well
+  as the `kubectl top` utility. These metrics are collected by
+  [metrics-server](https://github.com/kubernetes-incubator/metrics-server)
+  and are exposed via the `metrics.k8s.io` API. `metrics-server` discovers
+  all nodes on the cluster and queries each node's [Kubelet](/docs/admin/kubelet)
+  for CPU and memory usage. The Kubelet fetches the data from
+  [cAdvisor](https://github.com/google/cadvisor). `metrics-server` is a
+  lightweight short-term in-memory store.
```

Comment:
\o/ thank goodness for actual line wrapping -- wasn't sure if wrapping it myself was ok.  Had to turn off vim-pencil to prevent it from trying hard wrap the doc like this :-).